nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -4026,6 +4028,62 @@ impl DataStore {
         Ok(())
     }
 
+    /// Assert that all the region sets have three distinct targets
```

Comment:
This does not check for three distinct targets, but just for distinct targets, right?

Reply:
Correct, otherwise it fired for the fake volumes with only one target that the replacement machinery makes :)